app/src/main/java/org/kiwix/kiwixmobile/local_file_transfer/WifiDirectManager.kt

```diff
@@ -58,12 +58,12 @@ import javax.inject.Inject
  * Manager for the Wifi-P2p API, used in the local file transfer module
  */
 @SuppressWarnings("MissingPermission")
-class WifiDirectManager @Inject constructor(
+open class WifiDirectManager @Inject constructor(
```

Comment:
open?

Reply:
It complained about callbacks being protected instead of private and suggested the class should be open as an alternative. Will look into again..